src/minifluxapi.cpp

```diff
@@ -158,7 +185,9 @@ rsspp::Feed MinifluxApi::fetch_feed(const std::string& id, CurlHandle& cached_ha
 	feed.rss_version = rsspp::Feed::MINIFLUX_JSON;
 
 	const std::string query =
-		strprintf::fmt("/v1/feeds/%s/entries?order=published_at&direction=desc&limit=%u",
+		id == "starred" ? 
+			_("/v1/entries?starred=true")
```

Comment:
`_()` marks a string for translation. I doubt it's needed here.

Reply:
will do, I thought it was something else